server/metadata/sqlitegen/task_list_query_plan_test.go

```
@@ -0,0 +1,168 @@
+	requireQueryUsesAnyTableIndex(t, db, listWorkflowTaskListRows, "project_workflow_links", args...)
+	requireQueryUsesIndex(t, db, listWorkflowTaskListRows, "tasks_project_workflow_link_idx", args...)
+	requireQueryPlanDoesNotGroupIntoTemporaryTree(t, db, listWorkflowTaskListRows, args...)
+}
+
+func requireQueryPlanDoesNotGroupIntoTemporaryTree(t *testing.T, db *sql.DB, query string, args ...any) {
+	t.Helper()
+	rows, err := db.Query("EXPLAIN QUERY PLAN "+query, args...)
+	if err != nil {
+		t.Fatalf("explain query plan: %v", err)
+	}
+	defer rows.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rows.Close` is not checked

(errcheck)